packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/add.controller.js

```diff
@@ -123,15 +124,25 @@ export default class PciInstancesAddController {
     this.messages = this.messageHandler.getMessages();
   }
 
+  isBarreMetalBackup() {
+    return !!this?.backup.type.includes(BAREMETAL_LABEL);
+  }
+
   onPrivateNetworkChange(modelValue) {
     const networkId = get(modelValue, 'id');
+    this.publicNetwork = this?.publicNetworks.find((network) => {
```

Comment:
It doesn't seem necessary to use ? on **this**, maybe it's more relevant on **publicNetworks**